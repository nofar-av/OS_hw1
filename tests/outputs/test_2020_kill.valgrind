==26260== Memcheck, a memory error detector
==26260== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==26260== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==26260== Command: /home/omer/OS_wet1/smash
==26260== Parent PID: 26258
==26260== 
==26260== 
==26260== FILE DESCRIPTORS: 4 open at exit.
==26260== Open file descriptor 3: /home/omer/OS_wet1/tests/outputs/test_2020_kill.valgrind
==26260==    <inherited from parent>
==26260== 
==26260== Open file descriptor 2: /home/omer/OS_wet1/tests/outputs/test_2020_kill.err
==26260==    <inherited from parent>
==26260== 
==26260== Open file descriptor 1: /home/omer/OS_wet1/tests/outputs/test_2020_kill.out
==26260==    <inherited from parent>
==26260== 
==26260== Open file descriptor 0:
==26260==    <inherited from parent>
==26260== 
==26260== 
==26260== HEAP SUMMARY:
==26260==     in use at exit: 0 bytes in 0 blocks
==26260==   total heap usage: 323 allocs, 323 frees, 101,044 bytes allocated
==26260== 
==26260== All heap blocks were freed -- no leaks are possible
==26260== 
==26260== For counts of detected and suppressed errors, rerun with: -v
==26260== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
