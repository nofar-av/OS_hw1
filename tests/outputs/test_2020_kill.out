smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> smash> signal number 9 was sent to pid 5249
smash> smash> smash> smash> smash> [1] sleep 7& : 5487 1 secs
smash> smash> [1] sleep 7& : 5487 1 secs
[2] sleep 5& : 5532 0 secs
smash> smash> [1] sleep 7& : 5487 2 secs
[2] sleep 5& : 5532 1 secs
[3] sleep 3& : 5533 1 secs
smash> smash> smash> smash> smash> 